pkg/client/v1/model/NsALUA.go

```diff
@@ -0,0 +1,24 @@
+/**
+ * Copyright 2017 Hewlett Packard Enterprise Development LP
+ */
+
+package model
+
+
+/**
+ * <p>Golang package for NsALUA.</p>
+ */
+
+type NsALUA string 
```

Comment:
The naming pattern is off for all of these ns types.  They should follow the rest of the convention. 

ns_alua.go, ns_access_apply_to.go, etc.

Reply:
Sure, will fix it .